src/views/FormUser/FormUser.vue

```diff
@@ -0,0 +1,173 @@
+<script lang="ts" setup>
+import { ref, reactive } from 'vue';
+import { z } from 'zod';
+import { useRouter } from "vue-router";
+import Loading from '../../components/Loading.vue';
+
+console.log('start component user')
+
+const router = useRouter();
+
+interface MessageError {
+  name: string;
+  lastName: string;
+  email: string;
+};
+
+interface User {
+  name: string,
+  lastName: string,
+  email: string,
+};
+
+const schema = z.object({
+  lastName: z.string().min(1, 'Please fill in the Last Name.').regex(/^[A-Za-z][A-Za-z\s]*$/, "Only letters are allowed"),
+  email: z.string().email(),
+  name: z.string().min(1, 'Please fill in the name.').regex(/^[A-Za-z][A-Za-z\s]*$/, "Only letters are allowed")
+});
+
+const formData = ref<User>({
+  name: '',
+  lastName: '',
+  email: '',
+});
+
+const errorsDisplay = reactive<MessageError>({
+  name: '',
+  lastName: '',
+  email: ''
+});
+
+let loading = ref(false);
+
+const handleSubmit = async (): Promise<void> => {
```

Comment:
`handle` does not mean anything.
```suggestion
const onSubmit = async (): Promise<void> => {
```